gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -486,7 +486,13 @@ impl ObjectImpl for PravegaSink {
         match pspec.name() {
             PROPERTY_NAME_STREAM => {
                 let res = match value.get::<String>() {
-                    Ok(stream) => self.set_stream(&obj, Some(stream)),
+                    Ok(stream) => {
+                        if stream.is_empty() { 
+                            unreachable!("stream is empty")
```

Comment:
Is this really unreachable code? What if a user passes `pravegasink stream=""`? In this case, it should log an error and return. Unfortunately, there's no way to return an error from this function so `start()` will still be called. In any case, we do not want anything to panic due to user input errors.

Reply:
Changed.